app/dialogs/ManageActorDialog.js

```diff
@@ -324,11 +324,11 @@ define([
       if (!actor) {
         actor = {};
       }
-      var actorType = this.actortypes.value;
+      var actorType = parseInt(this.actortypes.value);
```

Comment:
is er geen mogelijkheid dat die value leeg is? deze doet mij daarvoor vrezen
> actor.type = {id: actorType || undefined };

anders best eerst checken alvorens te parseInten (ja dat is een werkwoord)

Reply:
Ik denk het niet eigenlijk. In dieje _reset wordt de default op de 0de value gezet, wat persoon is en als die actortypes niet bestaan gaat ge zowiezo niet mogen opslaan volgens mij. 

![image](https://user-images.githubusercontent.com/9083121/109984633-41ab0400-7d04-11eb-88a1-f1589d191851.png)

Reply:
ok, dan is die andere check gewoon uit automatisme wss